src/annotator/components/ContentInfoBanner.js

```diff
@@ -8,39 +15,108 @@ import { LabeledButton, Link } from '@hypothesis/frontend-shared';
  * A banner that displays information about the current document and the entity
  * that is providing access to it (eg. JSTOR).
  *
+ * layout columns:
```

Comment:
The layout in this component is relatively complex. I've attempted to comment thoroughly but let me know if anything feels confusing.